server/src/gameserver/sdk.rs

```diff
@@ -0,0 +1,173 @@
+use agones::Sdk;
+use tokio::{sync::mpsc, task};
+
+use std::collections::{HashMap, HashSet};
+
+use super::{BoardEvent, Error};
+
+use super::HealthChecker;
+
+pub struct Manager {
+    board_events: mpsc::Sender<BoardEvent>,
+    _health_checker: HealthChecker,
+}
+
+type BoardByUsers = HashMap<usize, HashSet<usize>>;
+
+impl Manager {
+    pub async fn setup() -> Result<Manager, Error> {
+        log::info!("connecting to agones sdk sidecar...");
+        let sdk = Sdk::new(None, None)
```

Comment:
Minor: Any reason `sdk` shouldn't be an attribute on `Manager` itself? then you won't have to keep passing `sdk` around.

Reply:
i think this could be a nice improvement - wdyt @muratozkan?

Reply:
yeah, I'll try it out

Reply:
@bhavaniMiro will merge this PR and do this separately so that everyone gets the fixes introduced here